app/UiTPAS/Jobs/ActivateUiTPASClientHandler.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\UiTPAS\Jobs;
+
+use App\Api\ClientCredentialsContext;
+use App\Domain\Integrations\Repositories\IntegrationRepository;
+use App\Domain\Integrations\Repositories\UdbOrganizerRepository;
+use App\Domain\Integrations\UdbOrganizerStatus;
+use App\UiTPAS\UiTPASApiInterface;
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+
+final class ActivateUiTPASClientHandler implements ShouldQueue
+{
+    use Queueable;
+
+    public function __construct(
+        private readonly UdbOrganizerRepository $udbOrganizerRepository,
+        private readonly IntegrationRepository $integrationRepository,
+        private readonly UiTPASApiInterface $api,
```

Comment:
I would make the name of the property more explicit

Reply:
class does not exist anymore, moved to ActivateUdbOrganizer, where I renamed the param